dashboard/test/ui/features/projects.feature

```diff
@@ -44,9 +44,8 @@ Scenario: Applab Flow
   And I wait to see "#footerDiv"
   Then I should see title "Code Ninja - App Lab"
   And element "#codeWorkspace" is hidden
-  And I press the first "#footerDiv .more-link" element
   And I make all links open in the current tab
-  And I press a button with xpath "//div[@id = 'footerDiv']//a[text() = 'How It Works']"
```

Comment:
If the "How it Works" menu option is still present for other app types, should we still test it?